onboarding/include/controller.h

```diff
@@ -12,7 +12,9 @@
 /* LED timer config */
 #define LED_TIMER_NAME          "led_timer"
 #define LED_TIMER_PERIOD        pdMS_TO_TICKS(2500)
+#define LIGHT_TIMER_PERIOD      pdMS_TO_TICKS(1000)
```

Comment:
To improve clarity, separate the led timer config from the light timer config. The comment above this section says `LED timer config`, so add the light timer stuff under a comment like `Light timer config`.

Reply:
By this, I mean something like:
```
/* LED timer config */
#define <All the led_timer-related config>

/* Light timer config */
#define <All the light_timer-related config>
```